client/verta/verta/registry/entities/_model.py

```diff
@@ -907,3 +913,23 @@ def delete(self):
         request_url = "{}://{}/api/v1/registry/registered_models/{}".format(self._conn.scheme, self._conn.socket, self.id)
         response = _utils.make_request("DELETE", request_url, self._conn)
         _utils.raise_for_http_error(response)
+
+    def set_action_type(self, action_type):
+        if not isinstance(action_type, action):
+            raise ValueError("action_type is not specified")
```

Comment:
Sorry I don't have a good guide for writing error messages 😭 

Here is my recommended wording, similar to [this other error message](https://github.com/VertaAI/modeldb/blob/dcfddae22341acf2cc95afda80c0d7ca08ba2aaf/client/verta/verta/client.py#L259-L262) in the client.

```suggestion
            raise ValueError(
                "`action_type` must be an object from verta.registry.action_type,"
                " not {}".format(type(action_type))
            )
```